hs_core/hydroshare/utils.py

```diff
@@ -1015,17 +985,6 @@ def resource_file_add_process(resource, files, user, extract_metadata=False,
                                                source_names=source_names, full_paths=full_paths,
                                                auto_aggregate=auto_aggregate, user=user)
     resource.refresh_from_db()
-    # receivers need to change the values of this dict if file validation fails
-    # in case of file validation failure it is assumed the resource type also deleted the file
-    file_validation_dict = {'are_files_valid': True, 'message': 'Files are valid'}
-    post_add_files_to_resource.send(sender=resource.__class__, files=files,
-                                    source_names=source_names,
-                                    resource=resource, user=user,
-                                    validate_files=file_validation_dict,
-                                    extract_metadata=extract_metadata,
-                                    res_files=resource_file_objects, **kwargs)
```

Comment:
If we are no longer sending the `post_add_files_to_resource` signal, we should "deregister" it here:
https://github.com/hydroshare/hydroshare/blob/5189-optimize-file-upload/hs_core/signals.py#L10

Reply:
Fixed